### ISSUE TITLE
Put stderr also into CMakeRun result

### ISSUE THREAD

**Issue by gzqx** [cdelledonne/vim-cmake#90]
If there is some problem while running the built binary, e.g. core dumped, currently the buffer will be blank and have no feedback.

**Comment by cdelledonne** [comment]
Hi, thanks for reporting this.

The `stderr` is actually part of the `:CMakeRun` result. The problem with fatal errors like segmentation faults is that the error message is not printed by the program itself, but by the shell running the program. See this post: https://unix.stackexchange.com/q/53289.

What we can do though is check the error code returned by the program and echo a message to the user if the program quit with a fatal error. I've added this small tweak, please try it from the latest `master`.

For instance, for a segmentation fault, you get this message on the command line:

```
[Vim-CMake] Executable was interrupted with fatal signal: SIGSEGV
```

**Comment by cdelledonne** [comment]
Closed in 4537f31cb6a483b6525d0836dac91352b57099d9
